recipe/build_base.sh

```diff
@@ -7,6 +7,19 @@ export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
 export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -std=c++17 -O3"
 export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
 
+if [[ $CONDA_TOOLCHAIN_BUILD != $CONDA_TOOLCHAIN_HOST ]]; then
+    # Conda does some swizzling when cross compiling, including moving
+    # the site-packages folder to the build prefix. So let's just
+    # manually add this to the compiler search path.
+    CPPFLAGS="-isystem ${BUILD_PREFIX}/lib/python$PY_VER/site-packages/numpy/core/include ${CPPFLAGS}"
+    CPPFLAGS="-isystem ${BUILD_PREFIX}/lib/python$PY_VER/site-packages/cairo/include ${CPPFLAGS}"
+    CPPFLAGS="-isystem ${BUILD_PREFIX}/include"
+    LDFLAGS="${LDFLAGS} -L${BUILD_PREFIX}/lib"
```

Comment:
You can remove these lines

Reply:
So only the `SP_DIR` line (below) is necessary?

Reply:
Yes. (I'm not even sure that is necessary. It's needed right now because `--with-numpy=$SP_DIR/numpy/core/include` is given. If that's removed too the `SP_DIR` line also not needed.)

Reply:
These lines ARE needed! The build was failing without them.

Reply:
@count0 Was the failure related to CGAL?  Now that our CGAL patch is fixed, are those lines needed?

Anyway, I already removed them.  If the build fails now, then presumably @isuruf will be interested to see why.

Reply:
To be clear, I am keeping the `SP_DIR` change, which you added in the same commit.  I only deleted the `CPPFLAGS` and `LDFLAGS` changes.

Reply:
Without these lines the build fails by not finding the pycairo and numpy includes...

Reply:
Indeed, the build failed ([log here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=511470&view=logs&jobId=385a3385-ae8b-54d1-9abe-3c614f74cb73&j=385a3385-ae8b-54d1-9abe-3c614f74cb73&t=d0913101-e169-5874-b37d-533167693e73)):

```
checking python module: numpy... yes
checking for numpy/arrayobject.h... yes
checking for cairomm-1.0... yes
checking python module: cairo... no
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/runner/miniforge3/conda-bld/graph-tool-suite_1653452601789/_build_env/venv/lib/python3.8/site-packages/cairo/__init__.py", line 1, in <module>
    from ._cairo import *  # noqa: F401,F403
ModuleNotFoundError: No module named 'cairo._cairo'
checking py3cairo.h usability... no
checking py3cairo.h presence... no
checking for py3cairo.h... no
configure: error: pycairo headers not found
```

Reply:
Yes, this was the first thing I had to fix.

Reply:
Okay, I'm just going to undo my most recent deletions via force-push.  That will put everything back into the last known passing state.